src/components/LightSwitch/index.tsx

```diff
@@ -0,0 +1,43 @@
+import { useTheme } from 'next-themes';
+import nookies from 'nookies';
+
+import { SECONDS_IN_MONTH } from '../../utils/constants';
+import { SunIcon } from './SunIcon';
+import { MoonIcon } from './MoonIcon';
+
+type LightSwitchProps = {
+  defaultTheme: string;
+};
+
+export const LightSwitch: React.FC<LightSwitchProps> = ({ defaultTheme }) => {
+  const { theme, setTheme } = useTheme();
+
+  const currentTheme = theme || defaultTheme;
+  const nextTheme = currentTheme === 'dark' ? 'light' : 'dark';
+
+  const toggleTheme = () => {
+    setTheme(nextTheme);
+    nookies.set(null, 'theme', nextTheme, {
+      path: '/',
+      maxAge: SECONDS_IN_MONTH,
```

Comment:
Do you think we need maxAge? Any reasoning behind it? I don't think there is "right or wrong" answer